BabelfishDump.spec

```diff
@@ -143,3 +144,14 @@ LD_LIBRARY_PATH=%{_builddir}/%{name}/src/interfaces/libpq $RPM_BUILD_ROOT/usr/bi
 %{_bindir}/bbf_dumpall
 
 %changelog
+* Tue Jan 16 2024 Rishabh Tanwar <ritanwar@amazon.com> - 16.1-2
+- Updated BabelfishDump RPM version to 16.1
+
+* Fri Dec 29 2023 Rishabh Tanwar <ritanwar@amazon.com> - 16.1-2
+- Do not dump default GRANTs
+	Skip dumping GRANTs between default Babelfish roles.
```

Comment:
The changelog format here is incorrect. From [the Fedora packaging guide](https://rpm-packaging-guide.github.io/#working-with-spec-files):
> Follow this format for the actual change entry:
> - Each change entry can contain multiple items - one for each change
> - Each item starts on a new line.
> - Each item begins with a - character.

That would make this entry:
```
* Fri Dec 29 2023 Rishabh Tanwar <ritanwar@amazon.com> - 16.1-2
- Do not dump default GRANTs
- Skip dumping GRANTs between default Babelfish roles.
```

Reply:
Yeah right, I was mostly trying to put extra context (commit subject) for each change seems like changelog should only be brief and include only commit title.